examples/lighting-app/silabs/efr32/BUILD.gn

```diff
@@ -61,8 +61,8 @@ declare_args() {
   # Argument to Disable IPv4 for wifi(rs911)
   chip_enable_wifi_ipv4 = false
 
-  # Argument to force enable WPA3 security on rs91x
-  rs91x_wpa3_only = false
+  # Argument to force enable WPA2/WPA3 security on rs91x
```

Comment:
rs91x_security_mode is checked against "wpa_mixed" also, shall we change to WPA/WPA2/WPA3 ?